autopush/router/simple.py

```diff
@@ -159,9 +160,19 @@ def route_notification(self, notification, uaid_data):
             returnValue(self.delivered_response(notification))
         else:
             self.metrics.increment("router.broadcast.miss")
-            returnValue(self.stored_response(notification))
-
-    #############################################################
+            retVal = self.stored_response(notification)
+            if self.udp is not None and "server" in self.conf:
+                # Attempt to send off the UDP wake request.
+                try:
+                    requests.post(self.conf["server"],
```

Comment:
Yes. Although I may have misunderstood the intent of a previous R comment. 
I'll add in a yield.